### PR TITLE
Instrumentation for RegressPhenotypes

### PR DESCRIPTION
**Pull request by nathanielparke**
Adds in instrumentation for RegressPhenotypes which can be ported to other section of code when the need arises.

### CODE REVIEW

**Comment by devin-petersohn** [comment]
pinging @fnothaft @heuermh to approve Jenkins build test.

**Comment by heuermh** [comment]
I was waiting on a successful build for #12 before adding other pull requests to Jenkins.

File: gnocchi-cli/src/main/scala/net/fnothaft/gnocchi/cli/RegressPhenotypes.scala [hunk 1]
```diff
@@ -116,10 +126,20 @@ class RegressPhenotypes(protected val args: RegressPhenotypesArgs) extends BDGSp
...
 
+    if (args.time) {
+      print_metrics_to_cmdline(metricsListener)
+    }
+
     // Log the results
     logResults(associations, sc)
   }
 
+  def print_metrics_to_cmdline(metricsListener: MetricsListener) {
```
Comment (heuermh) [inline gnocchi-cli/src/main/scala/net/fnothaft/gnocchi/cli/RegressPhenotypes.scala]: use camelCase for method names

File: gnocchi-cli/src/main/scala/net/fnothaft/gnocchi/cli/RegressPhenotypes.scala [hunk 2]
```diff
@@ -95,6 +97,9 @@ class RegressPhenotypesArgs extends Args4jBase {
 
   @Args4jOption(required = false, name = "-oneTwo", usage = "If cases are 1 and controls 2 instead of 0 and 1")
   var oneTwo = false
+
+  @Args4jOption(required = false, name = "-time", usage = "Prints out the timing information for RegressPhenotypes to commandline.")
```
Comment (heuermh) [inline gnocchi-cli/src/main/scala/net/fnothaft/gnocchi/cli/RegressPhenotypes.scala]: in ADAM we use `-print_metrics` for this, which comes from bdg-utils 

https://github.com/bigdatagenomics/utils/blob/master/utils-cli/src/main/scala/org/bdgenomics/utils/cli/Args4j.scala#L26

Or am I missing the use case here?

File: gnocchi-cli/src/main/scala/net/fnothaft/gnocchi/cli/RegressPhenotypes.scala [hunk 3]
```diff
@@ -107,6 +112,11 @@ class RegressPhenotypes(protected val args: RegressPhenotypesArgs) extends BDGSp
     val a = args.oneTwo
     //    println(s"\n\n\n\n\n\n oneTwo: $a \n\n\n\n\n\n\n\n")
 
+    import Timers._
+    Metrics.initialize(sc)
```
Comment (heuermh) [inline gnocchi-cli/src/main/scala/net/fnothaft/gnocchi/cli/RegressPhenotypes.scala]: this is also handled in bdg-utils for the `-print_metrics` argument

https://github.com/bigdatagenomics/utils/blob/master/utils-cli/src/main/scala/org/bdgenomics/utils/cli/BDGCommand.scala#L68

**Review by heuermh — CHANGES_REQUESTED** [review]

**Comment by heuermh** [comment]
Jenkins, add to whitelist

**Pull request closed without merging**